.github/scripts/tkn_check_parse.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# removes `actions's` default -e flag as it breaks the logic
+set +e
+
+setup() {
+    curl -LO "https://github.com/tektoncd/cli/releases/download/v${TKN_CLI_VERSION}/tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb" &>/dev/null
+    sudo dpkg -i ./tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb >/dev/null
+}
+
+is_yaml() {
+    FILE=`basename $1`
+    DIRNAME=`dirname $1`
+    [ ${DIRNAME##*/} == "workflows" ] && return 1
+    [ "${FILE#*.}" == "yaml" ] && return 0 || return 1
+}
+
+main() { 
+  TKN="tkn"
+  VER=`${TKN} version |head -1`
+  REGISTRY="foo.bar/foo:bar"
+  TMPFILE=$(mktemp)
+
+  echo -e "${VER}\n"
+  for file in ${CHANGED_FILES}; do
+      if is_yaml ${file}; then
+          echo -en "Checking file ${file}..."
+          cat ${file} | ${TKN} bundle push  ${REGISTRY} -f - >/dev/null 2> ${TMPFILE}
```

Comment:
Nit: Is there a reason for using `cat` instead of just `tkn bundle push -f $file`?  Also, do we need the `TKN` env var?

Reply:
I can change it. Also just copied my local `tkn` run that cats an output from a find. About the tkn, it was because I have more than one tkn locally to test. I will make it simpler.

Reply:
OK, thanks.